tests/huggingface/sagemaker_dlc_test.py

```diff
@@ -79,7 +77,7 @@ def run_test(args):
     pytest.param("bigscience/bloom-560m", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
     pytest.param("EleutherAI/gpt-neox-20b", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
     pytest.param("google/flan-t5-xxl", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
-    pytest.param("NousResearch/Llama-2-7b-chat-hf", None, "ml.inf2.48xlarge", marks=pytest.mark.inf2),
```

Comment:
I removed it because the startup time is too long, and it times-out from time to time. This should be solved in a future version hopefully. Using a smaller model means that:
- fetching weights from the hub is faster,
- formatting weights for neuron is faster (this step should be removed in the next version),
- loading weights on the neuron cores is faster (will be improved in the next version when formatting is removed).